co-simulation/bridge/carla_v0.10_xmlrpc_server.py

```diff
@@ -127,11 +166,18 @@ def _register_methods(self):
         self.server.register_function(self.get_sensor_data, 'get_sensor_data')
         self.server.register_function(self.get_detected_objects, 'get_detected_objects')
 
+        # Spectator / camera utilities
+        self.server.register_function(self.set_spectator_to_actor, 'set_spectator_to_actor')
+
         # Maps
         self.server.register_function(self.get_map_name, 'get_map_name')
         self.server.register_function(self.get_available_maps, 'get_available_maps')
         self.server.register_function(self.load_map, 'load_map')
 
+        # Coordinate transform configuration
+        self.server.register_function(self.set_input_frame_mode, 'set_input_frame_mode')
```

Comment:
How is this used in the ambassador side? Is there ever a case where the input frame isn't sumo?